streams/shared/src/main/scala/zio/stream/ZPipeline.scala

```diff
@@ -1820,22 +1819,23 @@ object ZPipeline extends ZPipelinePlatformSpecificConstructors {
         chunk => {
           val size = chunk.size
 
-          if (size == 0) reader
```

Comment:
In another PR, we were talking about the fact that this Chunk can probably never be empty